packages/@aws-cdk/aws-ses/lib/receipt-rule.ts

```diff
@@ -175,7 +175,7 @@ export class DropSpamReceiptRule extends CoreConstruct {
     super(scope, id);
 
     const fn = new lambda.SingletonFunction(this, 'Function', {
-      runtime: lambda.Runtime.NODEJS_10_X,
+      runtime: lambda.Runtime.NODEJS_12_X,
```

Comment:
This is not changing the test but actually changing the underlying custom resource. When customers upgrade, they're going to have their CF custom resource replaced. We'll need to make sure that the underlying handler is Node12 compatible.

Can we pull this change out and create a separate PR for these changes, for deeper scrutiny and customer notification?

Reply:
Reverted and created an issue instead https://github.com/aws/aws-cdk/issues/13223